sympy/integrals/prde.py

```diff
@@ -652,9 +702,11 @@ def param_rischDE(fa, fd, G, DE):
         # it will always terminate no matter what n is.
         n = bound_degree(a, b, r, DE, parametric=True)
     except NotImplementedError:
-        debug("param_rischDE: Proceeding with n = oo; may cause "
```

Comment:
Perhaps it should be replaced with a suitable comment. A temporary bound is set. Eventually, it will be removed.